### PR TITLE
Fixed the issue #19347.

### PR DESCRIPTION
**Pull request by tiagosampaio**
2.3 pull request: https://github.com/magento/magento2/pull/19393

After removing the Estimate Shipping Costs and Tax from cart using layout the totals were not displayed.

### Description (*)
Please check all the description in issue #19347.

### Fixed Issues (if relevant)

1. magento/magento2#19347: Removing "Estimate shipping costs and tax" from cart removes cart totals

### Manual testing scenarios (*)
The testing scenarios are described in the issue page.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @tiagosampaio. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Checkout/Block/Cart/Totals.php
Comment (sivaschenko) [inline app/code/Magento/Checkout/Block/Cart/Totals.php]: Please ensure the correct sequence of params in the phpdoc, it should correspond to the sequence of parameters in constructor

File: app/code/Magento/Checkout/Block/Cart/Totals.php
Comment (sivaschenko) [inline app/code/Magento/Checkout/Block/Cart/Totals.php]: Please change the access level to private

File: app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php
Comment (sivaschenko) [inline app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php]: Please use the serializer

File: app/code/Magento/Checkout/view/frontend/templates/cart/totals.phtml
Comment (sivaschenko) [inline app/code/Magento/Checkout/view/frontend/templates/cart/totals.phtml]: Can this javascript be moved to a js file?

File: app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php
Comment (sivaschenko) [inline app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php]: This class look more like a model than a block. Can you please move it to the Model namespace, also avoid `Helper` in class naming as it is too general, try to pick meaningful namespace and class name. Also, please add comment header for class and enable strict mode.

**Review by sivaschenko — CHANGES_REQUESTED** [review]

File: app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php
Comment (tiagosampaio) [reply app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php]: @sivaschenko done.

File: app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php
Comment (tiagosampaio) [reply app/code/Magento/Checkout/Block/Cart/Helper/ConfigProvider.php]: @sivaschenko done.

File: app/code/Magento/Checkout/Block/Cart/Totals.php
Comment (tiagosampaio) [reply app/code/Magento/Checkout/Block/Cart/Totals.php]: @sivaschenko done.

File: app/code/Magento/Checkout/Block/Cart/Totals.php
Comment (tiagosampaio) [reply app/code/Magento/Checkout/Block/Cart/Totals.php]: @sivaschenko done.

File: app/code/Magento/Checkout/view/frontend/templates/cart/totals.phtml
Comment (tiagosampaio) [reply app/code/Magento/Checkout/view/frontend/templates/cart/totals.phtml]: @sivaschenko this would demand other improvements to the code. This can be done in some other task. It's resolving the bug perfectly.

**Comment by tiagosampaio** [comment]
I'm still working on this.

**Comment by tiagosampaio** [comment]
Hey guys, please I need you to review this PR.

**Comment by sivaschenko** [comment]
This pull request should be on hold until the up port is merged

**Comment by VladimirZaets** [comment]
Hi @tiagosampaio. The branch `2.2-develop` will be closed for contribution from **July 15, 2019**. Please finish all your Pull Requests till this date, otherwise, they will be closed.
Thanks for collaboration.

**Comment by orlangur** [comment]
@VladimirZaets same here, it looks like https://github.com/magento/magento2/pull/19393 is stuck on @sivaschenko side.

**Comment by VladimirZaets** [comment]
Hi @tiagosampaio. 
The next 2.2 release will be the last of functional releases.
Due to it, we are closing 2.2-develop for contribution and all not ready for delivery Pull Requests. 
Thanks for collaboration.

**Pull request closed without merging**
